src/server.js

```diff
@@ -127,16 +132,11 @@ module.exports = function (classes) {
         var server = http.createServer();
 
         server.on('request', function onRequest(req, res) {
-          self.handleHttp(req, res)
-          .then(function(){
-          })
-          .catch(Error.ParseError, function(err){
-            Server.handleHttpError(req, res, err, self.opts.headers);
-          })
-          .catch(Error.InvalidRequest, function(err){
-            Server.handleHttpError(req, res, err, self.opts.headers);
-          })
-          ;
+          self.handleHttp(req, res).then(function () {
+          }).catch(function (err) {
```

Comment:
Well, I use the promise catch method. What you want to say is, catch specific types of errors, like before ? What if we miss one type ?

What are the types of errors that might appear and we want to send (and not send, to be specific and understand all) to the user (and so I can correct with the right cases)?